src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -110,21 +114,21 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach with TableDrivenPro
 
   def stubAWSRequests(
       inputBucket: String,
-      fileDownloadBytes: Option[Array[Byte]] = None,
+      tarFileName: String = "test.tar.gz",
```

Comment:
The changes in this method can be reverted as well because we don't need a different tar file.